main.go

```diff
@@ -66,13 +69,15 @@ func init() {
 	flag.Var(&secretsLabelSelector, "secrets-label-selector", "Label selector to find secrets to publish as metrics.")
 	flag.Var(&secretsAnnotationSelector, "secrets-annotation-selector", "Annotation selector to find secrets to publish as metrics.")
 	flag.StringVar(&secretsNamespace, "secrets-namespace", "", "Kubernetes namespace to list secrets.")
+	flag.StringVar(&secretsListOfNamespaces, "secrets-list-of-namespaces", "", "Kubernetes list of namespaces to search for secrets.")
```

Comment:
In the help text for both options let's mention that it's a "comma-delimited list" of namespaces.

Reply:
Done